### ISSUE TITLE
Timer too close, main MCU utilization high on BTT SKR E3 Turbo

### ISSUE THREAD

**Issue by henkyprayoga** [KevinOConnor/klipper#4422]
Just replaced main board into BTT SKR E3 Turbo. Fluidd graph shown that MCU host (Raspberry PI MCU) utilization is pretty high at that time.

![Screenshot 2021-06-25 235114](https://user-images.githubusercontent.com/862744/123459320-56433d00-d610-11eb-81b3-b062c65695e0.png)

[Uploading klippy-timer-too-close.log…]()

**Comment by henkyprayoga** [comment]
When I restarted klipper, klipper_mcu CPU usage is pretty high. I suspect it probably due to BMP680 sensor that I connected through the raspberry pi. Still, even after I disable BMP680, the CPU usage is pretty high at start. Is it normal? I'm running Raspberry PI 4B, 2G RAM, btw.

![Screenshot 2021-06-26 000229](https://user-images.githubusercontent.com/862744/123460629-1c733600-d612-11eb-9d1b-30f66751b303.png)
